bin/resources/GameIndex.yaml

```diff
@@ -3469,6 +3469,9 @@ SCES-52033:
         // Cop2 problems.
         patch=0,EE,003953F8,word,48438000
         patch=0,EE,003735FC,word,4B06521B
+        author=YukiXXL
+        //Speed Correction (25 FPS)
+        patch=1,EE,00175E1C,extended,24020019
```

Comment:
This patch is only going to modify the first byte, because in extended mode the first nibble is the size. 0 = 8bit 1 = 16bit 2 = 32bit, so only the 0x19 on the end will get written to memory, was that intended?

Reply:
alright there was no need to get defensive, my comment was related to the potential that the patch wasn't working correctly, but you explained that it was.

Reply:
And yes, I want you to change it for clarity's sake, please. The final effect will be identical, yes.

Reply:
> alright there was no need to get defensive, my comment was related to the potential that the patch wasn't working correctly, but you explained that it was.

no no lol i am not defensive at all, i'm just explaining it in more detail because i am afraid to have missed something or that my testing method could be wrong? so i wanted to lay it all out to you so you can confirm that i'm onto something real here, sorry for the confusion, thanks

Reply:
It sounds fine to me, I don't really know the game so I can't corroborate your findings and results, tbh the game felt fine to me at 50fps, but I don't really know it, as I say.

Reply:
> And yes, I want you to change it for clarity's sake, please. The final effect will be identical, yes.

done